pandas/tests/tslibs/test_timedeltas.py

```diff
@@ -16,6 +16,8 @@
         (1, 1),
         (np.int64(2), 2),
         (np.int32(3), 3),
+        (Timedelta(1e10), 1e10),
```

Comment:
I think this is worthy of its own test rather than parametrizing - can you separate this out?

Reply:
Sure, no problem. Is it useful to mark the test with a reference to the ticket?
Is this file OK? Or is there a better one?

Reply:
Totally - just add `# GH 32402` in the body of the test

Reply:
Done